config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "version": "0.64.23",
+    "version": "0.64.24",
```

Comment:
FYI the JSI change landed in 0.65, so RN 0.64 expects the v-table without drainMicroTasks

Reply:
I'm not sure how v8/JSI versions are handled. @tudorms, should we bump this version to `0.65.x` already?

Reply:
First we need to create a new branch, 0.64-stable, with the current state, then switch master to 0.65.1 along with the JSI changes.

Reply:
I have created branch `0.64-stable` based on the current `master`. I'll update the version on this PR accordingly.